pages/create-organization.tsx

```diff
@@ -140,17 +139,71 @@ function Index() {
       countryISO: "",
       countryISO3: "",
       name: ""
-    }
+    },
+    email: ""
   })
 
+  const [validationError, setValidationError] = React.useState(0);
+
+  function validateOrganizationDetails () {
+    if (organizationDetails.name === "" || 
+    organizationDetails.mission === "" || 
+    // organizationDetails.logo_url === "" ||  //TODO: figure out how to handle this validation
```

Comment:
Yeah, it should be just a string, but I noticed that sometimes the state is not set due some reason